docs/_posts/Meryem1425/2024-07-03-assertion_alcohol_smoking_general_symptoms_wip_en.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: model
+title: Detect Assertion Status from General Symptoms Entities
```

Comment:
Change the title to:
Detect Overdose and Withdrawal (assertion) status from Smoking-Alcohol related symptoms

Reply:
done